docs/GettingStarted.md

```diff
@@ -65,7 +65,8 @@ of the Datadog tracer, you can override the following defaults:
       debug: false,
       trace_agent_hostname: 'localhost',
       trace_agent_port: 8126,
-      env: Rails.env
+      env: Rails.env,
+      tags: {}
```

Comment:
As I said in https://github.com/DataDog/dd-trace-rb/pull/100 this is fine with me, providing what is the final value (if both are defined and different) needs be explicit in the docs.